keyboards/jc65/v32a/readme.md

```diff
@@ -5,13 +5,15 @@ JC65 ps2avrGB
 
 A 65% keyboard with RGB
 
-Keyboard Maintainer: [Jason Barnachea](https://github.com/nautxx)
+Keyboard Maintainer: [Jason Barnachea](https://github.com/nautxx)  
```

Comment:
Lines 8-12:

I think the desired markdown is space-space-line break, rather than space-space-line break-line break as you have it here. Shouldn't be any blank lines in between.